app/views/organizations/show.html.erb

```diff
@@ -1,49 +1,117 @@
 <%= render "individual_actions" %>
 
-<p>
-  <strong>Name:</strong>
-  <%= @organization.name %>
-</p>
+<div class="container">
 
-<p>
-  <strong>Description:</strong>
-  <%= @organization.description %>
-</p>
+<h2><%= @organization.name %></h2>
+<div class="organization-url"><%= @organization.url %></div>
 
-<p>
-  <strong>Address:</strong>
-  <span class="org-address"><%= @organization.address %></span>
-</p>
+<!-- column 1 -->
+  <div id="col1">
 
-<p>
-  <strong>Phone:</strong>
-  <% @organization.phones.each do |phone| %>
-    <%= phone %>
-  <% end %>
-</p>
+  <!-- Services -->
+    <div class="border-above">
+      <h4>Services</h4>
+        <ul>
+          <!-- Primary Services -->
+          <li>
+            <ul id="primary-services">
+            <li>Substance Abuse Treatment </li>
+            <li>Mental Health Treatment </li>
+            <li>Health & Wellness Education </li>
+          </ul>
+        </li>
+        <!-- Additional Services -->
+        <li>
+          <ul class="chunked">
+            <li id="communication">Internet, Phone, Voicemail</li>
```

Comment:
These list items all have ids. I'm wondering two things:
- What's the purpose of having them classified this way? Are we planning to style them differently based on type?
- If we do want to have them classified, perhaps class attributes would be more suitable?